client/src/GlobalStyles.tsx

```diff
@@ -6,7 +6,8 @@ export default createGlobalStyle`
         padding: 0;
         outline:0;
         box-sizing:border-box;
-        font-family: 'Open Sans', sans-serif; 
+        font-family: 'Open Sans', sans-serif;
+        font-size: 100%;
```

Comment:
This way, the font-size will always follow what's defined on the user's browser settings